Assignment_3.md

```diff
@@ -0,0 +1,243 @@
+package Core_Java_Assignments;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.ListIterator;
+import java.util.*;
+
+class Employee implements Serializable {
+
+    String Name;
+    String Email_Address;
+    String Age;
+    String Date_Of_Birth;
+
+    public Employee(String Name, String Email_Address, String Age, String Date_Of_Birth) {
+        this.Name = Name;
+        this.Email_Address = Email_Address;
+        this.Age = Age;
+        this.Date_Of_Birth = Date_Of_Birth;
+    }
+
+    public String toString() {
+        return "name: " + Name + "Email: " + Email_Address + "Age: " + Age + "Date Of Birth: " + Date_Of_Birth;
+    }
+
+}
+
+class Assignment_3_F {
+
+    static FileOutputStream fout;
+
+////////////////////////////////////////////...Add.../////////////////////////////
+
+    void add(ArrayList<Employee> array_list) {
+
+        Scanner sc1 = new Scanner(System.in);
+        int n = sc1.nextInt();
+        for (int i = 0; i <= n; i++) {
+            Scanner sc = new Scanner(System.in);
+
+            System.out.println("Pkease Enter Name..!");
+            String Name = sc.nextLine();
```

Comment:
same here please use lower camel case for variable name.

Reply:
yes sir